src/persist-client/src/batch.rs

```diff
@@ -486,6 +494,7 @@ impl BatchBuilderConfig {
             record_run_meta,
             expected_order,
             structured_key_lower_len: STRUCTURED_KEY_LOWER_LEN.get(value),
+            run_length_limit: MAX_RUN_LEN.get(value).clamp(2, usize::MAX),
```

Comment:
TIL `clamp`, cool

Reply:
Yeah... basically just `value.max(lower_bound).min(upper_bound)`, but I do find it easier to follow for whatever reason.